app/javascript/Pages/Notifications.jsx

```diff
@@ -1,25 +1,20 @@
 import React from "react";
-import Layout from "./Layout";
 import { Head, Link } from "@inertiajs/inertia-react";
-import "../stylesheets/body.css";
+import Layout from "./Layout";
 
 export default function Welcome(props) {
-  const token = document.querySelector(
-    "[name=csrf-token]"
-  ).content;
+  const token = document.querySelector("[name=csrf-token]").content;
```

Comment:
issue (blocking): use window variable and search for other places this is still being used